### PR TITLE
New Apache Hive Cheat Sheet

### LINKED ISSUE #3832: New Apache Hive Cheat Sheet
Cheat Sheet for Apache Hive   

------
IA Page:  http://duck.co/ia/view/apache_hive_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @Hasil-Sharma

### PR DESCRIPTION
**Pull request by Hasil-Sharma**
## Description of new Instant Answer, or changes

Cheat Sheet for Apache Hive Commands

## Related Issues and Discussions

Fixes #3832

---

Instant Answer Page: http://duck.co/ia/view/apache_hive_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/apache-hive.json [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        "sourceUrl": "https://www.tutorialspoint.com/hive/index.htm"
+    },
+    "aliases": [
+        "hive"
+    ],
+    "template_type": "terminal",
+    "section_order": [
+        "Retrieving Information",
+        "Table Joins",
+		"Partitioning",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/apache-hive.json]: Please fix the spaces here

**Comment by Hasil-Sharma** [comment]
@jophab : Made the changes ... Can we add something some test in `duckpan test CheatSheets` to throw some warning in case sheet json is not properly indented ?

**Comment by jophab** [comment]
@Hasil-Sharma I am not sure about the test you proposed even though it is a good idea. :+1: 
 We can easily avoid indentation problems by using json validator or formatter. 
DDG an IA for [Json Formatting and Validation](https://duckduckgo.com/?q=json+formatter&t=hs&ia=answer) by @sahildua2305

**Comment by Hasil-Sharma** [comment]
@jophab : I used @sahildua2305 's IA to make do. 
According to me incorporating a test for checking proper indentation and providing a command line (copy pasting the code back and forth vs. running a command) json validator or formatter will definitely be helpful. 

Thoughts ?

**Comment by tagawa** [comment]
Thank you for creating this cheatsheet @Hasil-Sharma and for the review @jophab! It looks good to me so merging, and it should go live in a few days time. 😄

**Comment by daxtheduck** [comment]
@Hasil-Sharma this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**